<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTadsConfDialog</class>
 <widget class="QDialog" name="QTadsConfDialog">
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Main Game Font</string>
           </property>
           <property name="buddy">
            <cstring>mainFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFontComboBox" name="mainFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="mainFontSizeComboBox"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Fixed-width Font</string>
           </property>
           <property name="buddy">
            <cstring>fixedFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFontComboBox" name="fixedFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="fontFilters">
              <set>QFontComboBox::MonospacedFonts</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fixedFontSizeComboBox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Serifed Font</string>
           </property>
           <property name="buddy">
            <cstring>serifFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFontComboBox" name="serifFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="serifFontSizeComboBox"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sa&amp;ns-Serif Font</string>
           </property>
           <property name="buddy">
            <cstring>sansFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QFontComboBox" name="sansFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sansFontSizeComboBox"/>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>S&amp;cript Font</string>
           </property>
           <property name="buddy">
            <cstring>scriptFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QFontComboBox" name="scriptFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="scriptFontSizeComboBox"/>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&amp;Typewriter Font</string>
           </property>
           <property name="buddy">
            <cstring>writerFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QFontComboBox" name="writerFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="fontFilters">
              <set>QFontComboBox::MonospacedFonts</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="writerFontSizeComboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Input Font</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QFontComboBox" name="inputFontBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="inputFontSizeComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="inputFontItalicCheckBox">
            <property name="text">
             <string>Italic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="inputFontBoldCheckBox">
            <property name="text">
             <string>Bold</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>inputFontBoldCheckBox</zorder>
         <zorder>inputFontItalicCheckBox</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Central Window</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Text</string>
              </property>
              <property name="buddy">
               <cstring>mainTextColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="mainTextColorButton"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Background</string>
              </property>
              <property name="buddy">
               <cstring>mainBgColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="mainBgColorButton"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Banner Windows</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Text</string>
              </property>
              <property name="buddy">
               <cstring>bannerTextColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="bannerTextColorButton"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Background</string>
              </property>
              <property name="buddy">
               <cstring>bannerBgColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="bannerBgColorButton"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Hyperlinks</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="underlineLinksCheckBox">
            <property name="text">
             <string>Underline Links</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="highlightLinksCheckBox">
            <property name="text">
             <string>Highlight when hovering</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Unvisited</string>
              </property>
              <property name="buddy">
               <cstring>linkUnvisitedColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="linkUnvisitedColorButton"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Hovering</string>
              </property>
              <property name="buddy">
               <cstring>linkHoveringColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="linkHoveringColorButton"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Clicked</string>
              </property>
              <property name="buddy">
               <cstring>linkClickedColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QToolButton" name="linkClickedColorButton"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Media</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>You can prevent games from displaying graphics or playing music and sound effects by clearing these checkboxes. If you turn off graphics and/or sounds, multimedia games will still work, but they'll be limited to displaying text.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="allowGraphicsCheckBox">
             <property name="text">
              <string>Allow Graphics</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="allowDigitalCheckBox">
             <property name="text">
              <string>Allow Digital Sound</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="allowMidiCheckBox">
             <property name="text">
              <string>Allow MIDI Sound</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="allowLinksCheckBox">
             <property name="text">
              <string>Allow Hyperlinks</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QTadsConfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QTadsConfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
